editor-meta.Rmd

```diff
@@ -26,4 +26,4 @@ The blog editors don't necessarily have write access to the blog-guidance repo b
 
 * Staff members can bypass review for tech notes but they can choose to request a review by blog editors.
 * Staff members' blog posts should be scheduled together with the Community Manager.
-* Staff members are responsible for creating the [discussion forum entry](#publish) for their own post and must [tweet from the rOpenSci Twitter account](#tweetpromo) (directly or via the Tweetdeck rOpenSci team) before tweeting from their personal account.
+* Staff members are responsible promoting their own posts from their personal account on Mastodon and/or LinkedIn tagging rOpenSci's accounts.
```

Comment:
```suggestion
* Staff members are responsible for promoting their own posts from their personal account on Mastodon and/or LinkedIn tagging rOpenSci's accounts.
```

However don't we expect staff members to use rOpenSci accounts first?

Reply:
There are not mutually exclusive, right?  
I will add this question on an issue so we don't delay the merge of this change.